spec/support/shared_examples/signable_configuration.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+RSpec.shared_context 'when signable is configured' do
+  Signable.configure do |config|
+    config.base_url = 'api.signable.co.uk'
+    config.api_key  = ENV['SIGNABLE_API_KEY']
```

Comment:
```suggestion
    config.api_key  = ENV.fetch('SIGNABLE_API_KEY')
```

Reply:
👎  If we do this, we lose the ability to run the test suite with simply:
`bundle exec rspec`
instead of:
`SIGNABLE_API_KEY='whatever' bundle exec rspec`
when there are existing VCR recordings.

If we use `ENV[` it won't raise an error when there is no `SIGNABLE_API_KEY`, and the tests will still run because Signable API requests won't actually be made because of the VCR recordings.

If I use `ENV.fetch` we'll always need `SIGNABLE_API_KEY='whatever'` in our test running commands or we'll get:
`KeyError: key not found: "SIGNABLE_API_KEY"`